Content.Server/Flash/FlashSystem.cs

```diff
@@ -195,12 +195,16 @@ private void OnFlashExamined(EntityUid uid, FlashComponent comp, ExaminedEvent a
 
         private void OnInventoryFlashAttempt(EntityUid uid, InventoryComponent component, FlashAttemptEvent args)
         {
-            // Forward the event to a worn helmet, if one is equipped.
-            if (_inventorySystem.TryGetSlotEntity(uid, "head", out var maskSlotEntity, component))
-                RaiseLocalEvent(maskSlotEntity.Value, args);
-            // Forward the event to the glasses, if any.
-            if(!args.Cancelled && _inventorySystem.TryGetSlotEntity(uid, "eyes", out var eyeSlotEntity, component))
-                RaiseLocalEvent(eyeSlotEntity.Value, args);
+            if(_inventorySystem.TryGetSlots(uid, out var slots))
+            {
+                foreach(var slot in slots)
+                {
```

Comment:
Think it's better being explicit slots IMO, also pass in inventorycomponent to trygetslots.